<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>232</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCheckBox" name="chb_shift">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>70</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>shift</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_command">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>210</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>210</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_menu">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>210</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>210</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>180</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>custom command</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_hotkey">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>20</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>70</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="maxLength">
    <number>1</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="chb_ctrl">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>270</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>70</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>ctrl</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>300</y>
     <width>140</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>140</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>hotkey</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_name">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>140</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>140</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>16777215</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_create">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>create</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmb_menu">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>94</y>
     <width>210</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>210</width>
     <height>20</height>
    </size>
   </property>
   <item>
    <property name="text">
     <string>mel file</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>python module</string>
    </property>
   </item>
  </widget>
  <widget class="QCheckBox" name="chb_alt">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>270</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>70</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>alt</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmb_command">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>210</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>210</width>
     <height>20</height>
    </size>
   </property>
   <item>
    <property name="text">
     <string>mel string</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>python string</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>python module</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="cmb_method">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>42</y>
     <width>210</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>210</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>180</width>
     <height>16777215</height>
    </size>
   </property>
   <item>
    <property name="text">
     <string>menu selected, command selected</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>menu selected, command custom</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>menu custom, command custom</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>menu preset, command preset</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>menu preset, command custom</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>68</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>180</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>custom menu</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>70</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="chb_command">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>run always after release</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
